lib/yardstick/measurement.rb

```diff
@@ -34,7 +34,7 @@ def initialize(rule)
     #
     # @api public
     def ok?
-      @result == true || skip?
+      @result.equal?(true) || skip?
```

Comment:
That is another proposed mutation.  I just wanted to make one type of change for this commit though